repo/ref/convert.go

```diff
@@ -39,7 +39,7 @@ func ConvertToVersionInfo(r *DatasetRef) dsref.VersionInfo {
 		build.CommitMessage = ds.Commit.Message
 	}
 	if ds != nil {
-		build.NumVersions = ds.NumVersions
+		build.CommitCount = ds.NumVersions
```

Comment:
Okay so we've changed NumVersions on verison info to CommitCount, but it's still NumVersions on the dataset.Dataset struct itself.

Now that collection is doing what it is supposed to be doing, do we need NumVersions on the dataset anymore? I assumed that was a bandaid until we got something like collection up and running.

Reply:
Would love to get rid of `dataset.NumVersions`. It was added way back when (Feb 2019) here: https://github.com/qri-io/dataset/pull/177, in order to support `qri list --num-versions`, but these newer pieces of tech like collections and VersionInfo are a better solution.

Reply:
filed issue https://github.com/qri-io/dataset/issues/267